### PR TITLE
Adds more copy changes

### PR DESCRIPTION
**Pull request by maxf**
Additionally, removes unused markup on history page
[Updates #106947502]

### CODE REVIEW

File: mtp_cashbook/templates/cashbook/includes/print_dialog.html [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-    <h3>{% trans "Before you print" %}</h3>
+    <h3>{% trans "Do you need to print?" %}</h3>
     {% blocktrans %}
-    <p>Have you tried changing the size of the window so you can see the payments and NOMIS at the same time?</p>
-    <p>You could also try using the ALT + TAB keys to switch between the windows.</p>
+    <p>Did you know:</p>
+    <ul>
+      <li>you can see the digital cashbook and NOMIS screens at the same time by changing the size of your windows</li>
+      <li>you can switch between windows using the ALT + TAB keys</li>
+      <li>you can copy and paste the prisoner number from one window to the other</li>
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/includes/print_dialog.html]: `you can copy and paste the prisoner number`… how? Can they use the Edit menu to do that? I thought it was disabled, and only the shortcut worked? @leighmoney / @madgoose?

File: mtp_cashbook/templates/cashbook/transaction_batch_list.html [hunk 2]
```diff
@@ -106,22 +105,14 @@ <h1 class="error-summary-heading heading-medium" id="error-summary-heading">{% t
 
     <dialog id="incomplete-batch-dialog" class="Dialog u-nojs-hidden" data-hide-close="true" data-disable-backdrop-close="true">
       <div class="Dialog-inner">
-        <h3>{% trans "Confirm submission" %}</h3>
-        <p>{% trans "Are you sure you want to submit only the selected payments as a batch?" %}</p>
-        <button type="submit" class="button" value="override">{% trans "Yes, submit" %}</button>
-        <button type="button" class="button button-secondary js-Dialog-close">{% trans "No, continue editing" %}</button>
+        <h3>{% trans "Are you sure?" %}</h3>
+        <p>{% trans "Do you want to submit only the selected credits?" %}</p>
+        <button type="submit" class="button" value="override">{% trans "Yes" %}</button>
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Would it be worth making the `Yes` button explain what it does? Something like `Submit selected credits`, perhaps, @leighmoney?

File: mtp_cashbook/templates/cashbook/includes/print_dialog.html (hunk 1 above)
Comment (maxf) [inline mtp_cashbook/templates/cashbook/includes/print_dialog.html]: tony sais: yes. they can do it

Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/includes/print_dialog.html]: :+1:

File: mtp_cashbook/templates/cashbook/transaction_batch_list.html (hunk 2 above)
Comment (maxf) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Tony: no, this is ok as is

**Pull request merged**